google-datacatalog-qlik-connector/tests/google/datacatalog_connectors/qlik/scrape/metadata_scraper_test.py

```diff
@@ -42,11 +41,18 @@ def test_constructor_should_set_instance_attributes(self):
         self.assertEqual('test-domain', attrs['_MetadataScraper__ad_domain'])
         self.assertEqual('test-username', attrs['_MetadataScraper__username'])
         self.assertEqual('test-password', attrs['_MetadataScraper__password'])
-        self.assertIsNotNone(attrs['_MetadataScraper__session'])
+
         self.assertIsNotNone(attrs['_MetadataScraper__qrs_api_helper'])
+        self.assertIsNotNone(attrs['_MetadataScraper__engine_api_helper'])
+
+    def test_constructor_should_not_set_session_related_attributes(self):
+        attrs = self.__scraper.__dict__
+
+        self.assertIsNone(attrs['_MetadataScraper__qrs_api_session'])
+        self.assertIsNone(attrs['_MetadataScraper__engine_api_auth_cookie'])
 
     @mock.patch(f'{__SCRAPER_MODULE}.authenticator.Authenticator')
-    def test_scrape_should_authenticate_user(self, mock_authenticator):
+    def test_scrape_qrs_api_should_authenticate_user(self, mock_authenticator):
```

Comment:
This test case name does not seem to be coherent with what is being tested.

From reading it, the main assertion seems to be that the scrape method correctly returns the scraped streams. Do you think it makes sense,  to change the test case name or change the assertions?

Reply:
I believe the method name it's correct. I've changed the assertions.